### PR TITLE
Add multiple validator

### PR DESCRIPTION
**Pull request by vermakhushboo**
Add a new validator that allows user to use multiple validators together.

### CODE REVIEW

File: src/Validator/Multiple.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    /**
+     * Get Type
+     *
+     * Returns validator type.
+     *
+     * @return string
+     */
+    public function getType(): string
```
Comment (eldadfux) [inline src/Validator/Multiple.php:98]: We might want to be able to control this from the constructor. I will help us generate better SDKs when set properly. The default value can stay mixed.

File: tests/Validator/MultipleTest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        // Valid text within length, but invalid URL
+        $this->assertFalse($this->validator->isValid('hello world'));
+
+        // Both conditions satisfied
+        $this->assertTrue($this->validator->isValid('http://example.com'));
+        $this->assertTrue($this->validator->isValid('https://google.com'));
+
+        // Neither condition satisfied
+        $this->assertFalse($this->validator->isValid('example.com/hello-world'));
+        $this->assertFalse($this->validator->isValid(''));
```
Comment (eldadfux) [inline tests/Validator/MultipleTest.php:30]: Let's also validate we get the correct description.

File: src/Validator/Multiple.php [hunk 3]
```diff
@@ -63,7 +66,7 @@ public function getDescription(): string
     {
         $description = '';
         foreach ($this->rules as $key => $rule) {
-            $description .= ++$key . '. ' . $rule->getDescription() . " \n";
+            $description .= ++$key . '. ' . $rule->getDescription() . " \\n";
```
Comment (eldadfux) [inline src/Validator/Multiple.php:69]: Extra `\`

File: src/Validator/Multiple.php [hunk 4]
```diff
@@ -18,25 +18,28 @@ class Multiple extends Validator
...
      * Multiple constructor can get any number of arguments containing Validator instances using PHP func_get_args function.
      *
      * Example:
      *
-     * $multiple = new Multiple($validator1, $validator2, $validator3);
+     * $multiple = new Multiple([$validator1, $validator2]);
+     * $multiple = new Multiple([$validator1, $validator2, $validator3], SELF::TYPE_STRING);
      */
-    public function __construct()
+    public function __construct(array $rules, ?string $type = null)
```
Comment (eldadfux) [inline src/Validator/Multiple.php:33]: Can we do this instead? not sure if the syntax is supported

```php
    public function __construct(array $rules, ?string $type = self::TYPE_MIXED)
```

If yes, we can drop the `if null` check

Comment (vermakhushboo) [reply src/Validator/Multiple.php:33]: Doesn't work

**Review by eldadfux — APPROVED** [review]

**Review by eldadfux — APPROVED** [review]

**Pull request merged**